tests/shared/commands.js

```diff
@@ -161,3 +161,9 @@ export const searchMockedTerm = (term) => {
   const regexTerm = createRegExp(term);
   return resultsFromDictionarySearch(regexTerm, term, mockedData);
 };
+
+export const getApiUrlRoute = (route = '/') => (
```

Comment:
nit: could you change this to `getAPIUrlRoute` so that it's consistent with `populateAPI`?